### PR TITLE
[10.0][FIX] module_auto_update: Don't set 'to upgrade' on void record…

### PR DESCRIPTION
**Pull request by rousseldenis**
…set modules

### CODE REVIEW

**Comment by rousseldenis** [comment]
@sbidoul

File: module_auto_update/models/module.py [hunk 1]
```diff
@@ -152,7 +152,9 @@ def upgrade_changed_checksum(self, overwrite_existing_translations=False):
         # exception, this would lead to corruption because
         # no upgrade would be performed and save_installed_checksums
         # would update cheksums for modules that have not been upgraded
-        ensure_module_state(self.env, changed_modules, 'to upgrade')
+        if changed_modules:
```
Comment (sbidoul) [inline module_auto_update/models/module.py]: Better put this test inside `ensure_module_state`.

**Comment by rousseldenis** [comment]
Of course! Done.

**Review by sbidoul — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]

**Comment by rousseldenis** [comment]
Pylint failed on changes that are not concerned by this.

**Review by lmignon — APPROVED** [review]
@rousseldenis pylint complains about new check... (rst syntax, ..)

**Comment by sbidoul** [comment]
rst syntax errors fixed at a91a75fe4e57bd63254cecbb7150724b3fde43cf

Merging. I will back and forward port.

**Pull request merged**